tests/unit/parse-test.js

```diff
@@ -9,7 +9,7 @@ describe('Parse', () => {
     describe('literals', () => {
       it('handles language subtags', () => {
         let base = 'https://www.wikidata.org/wiki/Special:EntityData/Q2005.ttl'
-        let mimeType = 'text/turtle'
+        let mimeType = 'text/turtle; charset="UTF-8"'
```

Comment:
this tests verifies that parsing *handles language subtags*, i suggest creating another *it* to verify proper handling of mimeType